### PR TITLE
Select & dropdown

### PR DESCRIPTION
**Pull request by Benedito0**
| - | - |
| --- | --- |
| Issue | https://github.com/ripe-tech/ripe-white/issues/144 |
| Decisions |   |
| Animated GIF |aMx87JEvuG |

### CODE REVIEW

File: vue/components/dropdown/dropdown.vue [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+.dropdown > .dropdown-item.highlighted {
+    background-color: #272727;
+    color: #ffffff;
+}
+
+.dropdown > .dropdown-item.selected {
+    background: url("~./assets/ic-check.svg") right center no-repeat;
+    background-color: #1d1d1d;
+    color: #ffffff;
+}
```
Comment (3rdvision) [inline vue/components/dropdown/dropdown.vue]: add
```
background-size: 14px;
background-position: right 12px center;
```
and tune the sizes if you like

**Comment by gcandal** [comment]
@Benedito0 @3rdvision : https://github.com/ripe-tech/ripe-commons-pluginus/pull/75/commits/aedebadca7320f1202dbd684f746d9e6709d7e7c
- We can't have styles specific to some use-cases in the components. That should be defined by the parent, who knows the context
- There was a border missing, which led to a missing focus indicator

**Comment by gcandal** [comment]
@Benedito0 please move this to RIPE White using the new component structure.

**Comment by joamag** [comment]
We need to migrate this to RIPE White

**Comment by joamag** [comment]
Closing this as this code has been merged directly to RIPE White

**Pull request closed without merging**